Model/Order/Item/Builder.php

```diff
@@ -170,14 +182,12 @@ public function buildSkuId(Item $item)
             //An item with bundle product and group product may have more than 1 child.
             //But configurable product item should have max 1 child item.
             //Here we check the size of children, return only if the size is 1
-            if (count($children) == 1
-                && array_key_exists(0, $children)
+            /** @var Item[] $children */
+            if (array_key_exists(0, $children) && $children[0] instanceof Item
```

Comment:
The second condition should be on own line as well

Reply:
Fixed